client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -321,7 +334,9 @@ export class CheckoutThankYouHeader extends PureComponent {
 			isDomainTransfer( primaryPurchase ) ||
 			isSiteRedirect( primaryPurchase )
 		) {
-			return translate( 'Manage domain' );
+			return this.isSingleDomainPurchase()
+				? translate( 'Manage domain' )
+				: translate( 'Manage domains' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Manage my domains' )` *ES Score: 7*